src/operation/get-callback-examples.ts

```diff
@@ -18,7 +18,6 @@ export default function getCallbackExamples(operation: RMOAS.OperationObject) {
   const ret: CallbackExamples = [];
 
   // spreads the contents of the map for each callback so there's not nested arrays returned
-  // eslint-disable-next-line sonarjs/no-empty-collection
```

Comment:
I've removed `eslint-plugin-sonarjs` from our config because it doesn't support ESLint 8 yet. TBD if it'll come back because it hasn't really offered us much.